torsion.json

```diff
@@ -2,3211 +2,3853 @@
   {
     "molecule_ID": "mobley_1017962",
     "only_aromatic_rings": true,
-    "num_torsions": 5
+    "num_rotatable_bonds": 4,
+    "num_aliphatic_bonds": 0
```

Comment:
Can we maybe name these `num_acyclic_rotatable_bonds` and `num_aliphatic_ring_rotatable_bonds`? They are a bit longer but I find them less ambiguous.